collection-scripts/version

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+function version() {
+  # get version from odh/rhods operator image
+  version=''
+  operator_name=$(oc get pod -n redhat-ods-operator --ignore-not-found |grep rhods-operator |awk '{print $1}')
+
+  [[ z != z"${operator_name}" ]] && version=$(oc exec ${operator_name} -n redhat-ods-operator -c rhods-operator -- env|grep OPERATOR_CONDITION_NAME|awk -F'=' '{print $2}'|     sed -n -r -e 's/.*([[:digit:]]+\.[[:digit:]]+\.[[:digit:]]+(:?\.[[:digit:]])?(:?-[^@]+)?).*/\1/p')
```

Comment:
@VaishnaviHire please review this script because I assume that the OPERATOR_CONDITION_NAME is always set in RHODS operator image container

Reply:
@Jooho I think we should try to get version from the CSV first. This function might fail when for some reason operator pod is down

Reply:
@VaishnaviHire makes sense, I changed the order.